### PR TITLE
skip null round retry

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Skip retry if the height to query is a null round. This way we can advance the block height in fendermint faster.

### CODE REVIEW

File: fendermint/vm/topdown/src/finality/fetch.rs [hunk 1]
```diff
@@ -34,6 +34,12 @@ macro_rules! retry {
                 tracing::warn!(
                     "cannot query ipc parent_client due to: {e}, retires: {retries}, wait: {wait:?}"
                 );
+
+                // there is no point in retrying if the current block is null round
+                if is_null_round_error(&e) {
```
Comment (aakoshh) [inline fendermint/vm/topdown/src/finality/fetch.rs:39]: Should this be moved above the warn, so it doesn't suggest there will be retries?

Comment (cryptoAtwill) [reply fendermint/vm/topdown/src/finality/fetch.rs:35]: I shifted the logging

**Review by aakoshh — APPROVED** [review]
Cheers :pray:

**Review by adlrocha — APPROVED** [review]
@cryptoAtwill, do you mind fixing the CI before merging? Thanks!

**Pull request merged**